toml.md

```diff
@@ -614,9 +628,10 @@ contributors = [
 ]
 ```
 
-Arrays can span multiple lines. A terminating comma (also called trailing comma)
-is permitted after the last value of the array. Any number of newlines and
-comments may precede values, commas, and the closing bracket.
+Arrays can span multiple lines. A terminating comma (also called a trailing
+comma) is permitted after the last value of the array. Any number of newlines
+and comments may precede values, commas, and the closing bracket. Indentation
+between array values and commas is treated as whitespace, and is ignored.
```

Comment:
This can be simplified to "... treated as whitespace and ignored."

Reply:
Sounds good.